docs/release_notes.adoc

```diff
@@ -3,6 +3,19 @@ include::include.adoc[]
 
 == 2.0-M3 (tbd)
 
+=== Breaking Changes
+
+==== New meaning of `>> _`
+The meaning of `>> \_` has changed form "use the default response" to "return a stubbed value" (<<interaction_based_testing.adoc#_returning_a_default_response,Docs>>).
+The original behavior was only ever documented in the Javadocs and was basically the same to just omitting it.
+With the new behavior, you can have a `Mock` or `Spy` return the same value as a `Stub`.
+This is also more in line with the fact that `_` in arguments constraints stands for any non-`null` value, while a `Mock` returns `null` by default.
```

Comment:
This is either wrong or the documentation has a bug.
Docs say:
> 1 * subscriber.receive(\_)              // any single argument (including null)
> 1 * subscriber.receive(*_)             // any argument list (including the empty argument list)
> 1 * subscriber.receive(!null)          // any non-null argument

Reply:
yeah its wrong